docs/sources/reference/run.md

```diff
@@ -552,6 +553,20 @@ would be 2*300M, so processes can use 300M swap memory as well.
 We set both memory and swap memory, so the processes in the container can use
 300M memory and 700M swap memory.
 
+We can set oom control in which whether to disable OOM Killer or not when OOM
+happened. It is strongly recommended that use this option with `-m`.
+
+Examples:
+
+    $ docker run -ti -m 100M oom-kill-disable=true ubuntu:14.04 /bin/bash
+
+This example limit the memory to 100M and disable the OOM Killer to this container.
+
+    $ docker run -ti oom-kill-disable=true ubuntu:14.04 /bin/bash
+
+This example is dangerous for we do not limit the memory, if this container is keep
```

Comment:
> This example is dangerous; there is no memory-limit set for the container and the OOM-killer is disabled. This may result in the host running out of memory, and system processes being killed to free memory.